src/ol_concourse/lib/resources.py

```diff
@@ -91,14 +91,18 @@ def github_issues(  # noqa: PLR0913
     name: Identifier,
     repository: str,
     issue_prefix: str,
+    auth_method: Literal["token", "app"] = "token",
```

Comment:
Should we just default this to "app" rather than having to override it explicitly in other places?

Reply:
Maybe? I was operating on the "don't break what is currently running" mindset but I'm not sure that applies here.